decentraland/explorer-website
# feat: add custom catalyst warning

This PR warn any user that try to connect with a custom catalyst

<img width="1020" alt="Screen Shot 2022-08-05 at 01 08 41" src="https://user-images.githubusercontent.com/208789/182975491-10d50c4c-7295-4ba4-acd9-346bedd3add3.png">

# How to test it

- Open the link
- Add the `CATALYST=peer.decentraland.org` as query string
- you should be warn with a modal
